src/Microsoft.AspNet.Server.Kestrel/Http/Frame.cs

```diff
@@ -146,7 +185,8 @@ public void Reset()
 
             _prepareRequest?.Invoke(this);
 
-            _requestAbortCts?.Dispose();
+            _manuallySetRequestAbortToken = null;
+            _abortedCts = null;
```

Comment:
No.  CancellationTokenSource.Dispose does a few things:
- Disposes of the ManualResetEvent that's lazily-allocated when the WaitHandle is accessed
- Disposes of the Timer that's created if the CT's time-based ctor or its CancelAfter method is used
- Disposes of the linked registrations created from a CTS created with CreateLinkedTokenSource

The third one isn't relevant as this CTS isn't being created with CreateLinkedTokenSource.
The second one isn't relevant as we're not using timers.
And while the third one is possibly relevant, it's a) rare that code these days uses CancellationToken.WaitHandle, and b) if it does get lazily-allocated, it'll still get cleaned up via the finalizer.

Given all that, it didn't seem worthwhile to Dispose of it and have to have other code that worried about delayed with the resulting race conditions on its use.